content/400-reference/300-database-reference/03-supported-databases.mdx

```diff
@@ -43,11 +45,11 @@ Prisma currently supports the following databases.
 | CockroachDB-as-a-Service     | \*      |
 | MongoDB Atlas                | \*      |
 
-Note that a fixed version of SQLite is shipped with every Prisma release.
-
-An asterisk (\*) indicates that the version number is not relevant; either all versions are supported, there is not a public version number, etc.
 
 &sup1; This does not include support for [Data API for Aurora Serverless](https://github.com/prisma/prisma/issues/1964).
 
+---
```

Comment:
Seems this actually escapes out of the content area on rendering - if we ever want to use `---` we should get that fixed.